src/constants.js

```diff
@@ -361,3 +361,6 @@ export const DCB_VIRTUAL_USER = {
   },
   type: DCB,
 };
+
+export const PROFILE_PIC_DEFAULT_MAX_SIZE = 10;
```

Comment:
Dimensionless constants are difficult to understand. Is this size in bytes, terabytes, pixels wide, pixels high? It's common to include the dimension right in the name, e.g. `PROFILE_PICTURE_MAX_SIZE_IN_KB = 10`. At the very least include a comment.

Reply:
Hey @zburke sorry I missed this comment earlier. This constant name change is addressed in follow-up PR -https://github.com/folio-org/ui-users/pull/2693